docs/typo3-configuration.md

```diff
@@ -0,0 +1,28 @@
+# TYPO3 configuration
+
+## AdditionalConfiguration.php example
+
+```php
+if ($_SERVER['TYPO3_CONTEXT'] === 'Development/docker') {
+    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['host'] = getenv('typo3DatabaseHost') ?: 'global-db';
+    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['port'] = getenv('typo3DatabasePort') ?: '3306';
+    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['user'] = getenv('typo3DatabaseUsername') ?: 'root';
+    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['password'] = getenv('typo3DatabasePassword') ?: 'root';
+    $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections']['Default']['dbname'] = getenv('typo3DatabaseName') ?: 'default_database';
+
+    
+    $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport'] = 'smtp';
+    $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_encrypt'] = '';
+    $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_username'] = '';
+    $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_password'] = '';
+    $GLOBALS['TYPO3_CONF_VARS']['MAIL']['transport_smtp_server'] = getenv('SMTP_MAIL_SERVER') ?: 'global-mail:1025';
```

Comment:
as we now have POSTFIX_RELAYHOST documented and as Default we can remove this line.
```suggestion
```

Reply:
we need to update the reset of ``$GLOBALS['TYPO3_CONF_VARS']['MAIL']`` as well.

Reply:
I would not throw out TYPO3 mail.

And I have an update for a more elegant method:
https://github.com/pluswerk/php-dev/pull/20

Reply:
Resolved: Use mail method or smtp